modules/manage/pages/topic-iceberg-integration.adoc

```diff
@@ -0,0 +1,439 @@
+
+. Set the `iceberg_enabled` configuration option on your cluster to `true`. You must restart your cluster if you change this configuration for a running cluster. 
++
+[,bash]
+----
+rpk cluster config set iceberg_enabled true 
+----
++
+[,bash,role=no-copy]
+----
+Successfully updated configuration. New configuration version is 2.
+----
+
+. Create a new topic.
++
+[,bash,]
+----
+rpk topic create <new-topic-name> --partitions 1 --replicas 1
+----
++
+[,bash,role=no-copy]
+----
+TOPIC            STATUS
+new-topic-name   OK
+----
+
+. Enable the integration for the topic by configuring `redpanda.iceberg.mode`. You can choose one of the following modes:
++
+--
+* `key_value`: Creates an Iceberg table using a simple schema, consisting of two columns, one for the record metadata including the key, and another binary column for the record's value.
+* `value_schema_id_prefix`: Creates an Iceberg table whose structure matches the Redpanda schema for this topic, with columns corresponding to each field. You must register a schema in the Schema Registry (see next step), and producers must write to the topic using the Schema Registry wire format. Redpanda parses the schema used by the record based on the schema ID encoded in the payload header, and stores the topic values in the corresponding table columns. 
+* `disabled` (default): Disables writing to an Iceberg table for this topic.
+--
++
+[,bash]
+----
+rpk topic alter-config <new-topic-name> --set redpanda.iceberg.mode=<topic-iceberg-mode>
+----
++
+[,bash,role=no-copy]
+----
+TOPIC            STATUS
+new-topic-name   OK
+----
+
+. Register a schema for the topic. This step is required for the `value_schema_id_prefix` mode, but is optional otherwise.
++
+[,bash]
+----
+rpk registry schema create <subject-name> --schema </path-to-schema> --type <format>
+----
++
+[,bash,role=no-copy]
+----
+SUBJECT                VERSION   ID   TYPE
+new-topic-name-value   1         1    PROTOBUF
+----
+
+The Iceberg table is inside a namespace called `redpanda`, and has the same name as the Redpanda topic name. As you produce records to the topic, the data also becomes available in object storage for consumption by Iceberg-compatible clients. 
+
+== Schema support and mapping
+
+The `redpanda.iceberg.mode` property determines how Redpanda maps the topic data to the Iceberg table structure. You can either have the generated Iceberg table match the stucture of a Avro or Protobuf schema in the Schema Registry, or use the `key_value` mode where Redpanda stores the record values as-is in the table. 
+
+The JSON Schema format is not supported in this beta release. If your topic data is in JSON, it is recommended to use the `key_value` mode.
```

Comment:
```suggestion
The JSON Schema format is not supported in this release. If your topic data is in JSON, it is recommended to use the `key_value` mode.
```

Remove this as we're going to GA this version

Reply:
@Deflaimun Iceberg will continue to be in beta throughout 24.3 GA